### PR TITLE
Change headers to Andes

### PR DESCRIPTION
**Pull request by thomasdewulf**

### CODE REVIEW

File: components/Profile/Profile.tsx [hunk 1]
```diff
@@ -17,14 +17,14 @@ const Profile: React.FC<ProfileProps> = (props: ProfileProps) => {
 
   return (
     <View style={styles.container}>
-      <Text
+      <HeaderText
         style={[
           styles.greetingHeader,
           { color: Colors[colorScheme].headerColor },
```
Comment (NuclearCookie) [inline components/Profile/Profile.tsx:23]: No need for this color style if you use HeaderText

**Review by NuclearCookie — CHANGES_REQUESTED** [review]

File: components/Profile/Profile.tsx (hunk 1 above)
Comment (thomasdewulf) [reply components/Profile/Profile.tsx]: Fixed!

File: components/Profile/Profile.tsx [hunk 2]
```diff
@@ -34,14 +29,7 @@ const Profile: React.FC<ProfileProps> = (props: ProfileProps) => {
...
         <View style={styles.activityContainer}>
-          <Text
-            style={[
-              styles.activityHeader,
-              { color: Colors[colorScheme].headerColor },
-            ]}
-          >
-            Workshopkeuze
-          </Text>
+          <Text style={[styles.activityHeader]}>Workshopkeuze</Text>
```
Comment (NuclearCookie) [inline components/Profile/Profile.tsx:32]: This isn't using HeaderText so it won't be blue. Confirm if this is what you want :)

**Review by NuclearCookie — CHANGES_REQUESTED** [review]

File: components/Profile/Profile.tsx (hunk 2 above)
Comment (thomasdewulf) [reply components/Profile/Profile.tsx:32]: Kben ni goe bezig haha

**Pull request merged**